imjoy/core/services.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import sys
+
+logging.basicConfig(stream=sys.stdout)
```

Comment:
We should probably set up the logging config only once when launching.

Reply:
You mean a global logger?

Reply:
The logging only needs to be configured once for the duration of the application runtime. Somewhere in the setup phase.

Each module can have a logger but we can set the logging config centrally for the application.

Reply:
This is something for a future PR. Not this one.

Reply:
Sounds good!